### DISCUSSION TITLE
Energenie board support - multiple devices

### DISCUSSION THREAD

**Discussion by megatron-uk** [gpiozero/gpiozero#936] [Q&A]
Hi, I'm looking for an automation solution to control _many_ Energenie power strips (https://energenie4u.co.uk/catalogue/product/ENER010). 

Has anyone successfully used the Energenie board support (https://gpiozero.readthedocs.io/en/v1.2.0/api_boards.html#energenie) to pair with and control more than one device? Specifically I've got 5 or 6 of these power strips in my gaming room (with a corresponding amount of remote controls!) and I'm looking to both increase the number of strips and reduce the amount of remotes.

Although, actually, having looked at their docs for the Pi board they supply, it appears to be hardcoded to support only 4 discrete channels (excluding all on/all off):
 - https://energenie4u.co.uk/res/pdfs/ENER314%20UM.pdf

**Comment by lurch** [comment]
I don't have any of the Energenie power strips you mentioned, but if the Energenie Pi board is indeed limited to controlling just 4 sockets, I guess in a worst-case scenario you could use multiple Pis with multiple Energenie Pi boards, and control them all from a single Pi using GpioZero's remote GPIO feature? https://gpiozero.readthedocs.io/en/stable/remote_gpio.html

@bennuttall @waveform80 Or would https://github.com/RPi-Distro/python-gpiozero/blob/master/gpiozero/boards.py#L1966 currently prevent multiple Energenie's working with remote GPIO? :man_shrugging:
